schemas/core/itinerary.json

```diff
@@ -10,8 +10,7 @@
       "$ref": "components/units.json#/definitions/uuid"
     },
     "isOvertaken": {
-      "type": "boolean",
-      "default": false
```

Comment:
This should not have default properties. Need to check with clients but pretty sure they use it as optional prop